static/js/publisher/release/components/releasesTableCell.js

```diff
@@ -283,10 +285,24 @@ const ReleasesTableCell = props => {
     props.undoRelease(revision, channel);
   }
 
+  const [isHovered, setIsHovered] = useState(false);
+  const onMouseEnter = () => {
+    setIsHovered(true);
+    if (buildId) {
+      setHoveredBuild(buildId);
+    }
+  };
+
+  const onMouseLeave = () => {
+    setIsHovered(false);
+    setHoveredBuild(null);
+  };
+
   const className = [
     "p-releases-table__cell",
     isUnassigned ? "is-unassigned" : "",
     isActive ? "is-active" : "",
+    isHovered || (buildId && buildId === hoveredBuild) ? "is-hovered" : "",
```

Comment:
`hoveredBuild` is not required, because it can be null (if hovering revision that is not part of any build set or not hovering any revision).
In such case this shouldn't fail, because `null` won't equal `buildId`. And if `buildId` is null as well the first part (`buildId && ....`) will be false.

Reply:
How does JavaScript work again?